### DISCUSSION TITLE
「secioss_ldif」フォルダについて

### DISCUSSION THREAD

**Discussion by ai-sato01** [SeciossOpenSource/LISM#1] [Q&A]
LISM用OpenLDAPサーバの設定
LISMの管理用LDAPサーバとして利用するための設定を行います。
パッケージファイル内にある「secioss_ldif」フォルダ以下をOpneLDAPをインストールしたサーバにコピー（ディレクトリは何処でも構いません）して下さい。
「secioss_ldif」フォルダ内には5つのldifファイルがあります。このうち4つのファイルを編集し、LDAPの初期設定を行います。

⇒Wiki内に上記記載がありますが「secioss_ldif」フォルダが見つかりません。

**Comment by kaoru-sekiguchi** [comment]
packages/のLISM-4.1.3-1.x86_64.tar.gzを展開すると、展開したフォルダ内にsecioss_ldifというフォルダがありますので、そちらのLDIFファイルをご利用ください。

**Reply by ai-sato01** [reply]
回答ありがとうございます。
ファイルを確認することができました。
